draft-ietf-quic-multipath.md

```diff
@@ -223,31 +223,20 @@ the use of the multipath extension during the connection handshake,
 as specified in {{QUIC-TRANSPORT}}. The new transport parameter is
 defined as follows:
 
-- name: enable_multipath (current version uses 0x0f739bbc1b666d04)
-- value: 0 (default) for disabled.
-
-The valid options for the value field are listed in
-{{param_value_definition}}:
-
-Option | Definition
----------|---------------------------------------
-0x0      | don't support multipath
-0x1      | supports multipath as defined in this document
-{: #param_value_definition title="Available value for enable_multipath"}
-
-If for any one of the endpoints, the parameter is absent or set to 0,
-the endpoints MUST fallback to
-{{QUIC-TRANSPORT}} with single active path and MUST NOT use any frame or
+- enable_multipath (current version uses 0x0f739bbc1b666d05): the
+  enable_multipath transport parameter is included if the endpoint supports
+  the multipath extension as defined in this document. This parameter has
+  a zero-length value.
+
+If any of the endpoints does not advertise the enable_multipath transport
+parameter, then the endpoints MUST fallback to {{QUIC-TRANSPORT}} with
+a single active path and MUST NOT use any frame or
```

Comment:
```suggestion
parameter, then the endpoints MUST NOT use any frame or
```

I think "MUST NOT fallback to .. with a single active path" might be excessive.

Consider the case of having another extension that uses multiple active paths. With the absence of `enable_multipath` transport parameter on either side, do we want to prohibit use of such extension?

I think we do not need to.

Reply:
Fair point.